docs/configure-custom-values-envoy.md

```diff
@@ -86,42 +80,21 @@ You can configure them using the same syntax as the affinity of Kubernetes. So,
 ```yaml
 envoy:
   affinity:
-    nodeAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        nodeSelectorTerms:
-          - matchExpressions:
-              - key: scalar-labs.com/dedicated-node
-                operator: In
-                values:
-                  - scalardb
     podAntiAffinity:
-      requiredDuringSchedulingIgnoredDuringExecution:
-        - labelSelector:
-            matchExpressions:
-              - key: app.kubernetes.io/name
-                operator: In
-                values:
-                  - scalardb
-              - key: app.kubernetes.io/app
-                operator: In
-                values:
-                  - envoy
-          topologyKey: kubernetes.io/hostname
-```
-
-### Taints/Tolerations configurations (Recommended in the production environment)
-
-If you want to control pod deployment using the taints and tolerations of Kubernetes, you can use `envoy.tolerations`.
-
-You can configure them using the same syntax as the tolerations of Kubernetes. So, please refer to the official document [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for more details on the tolerations configuration of Kubernetes.
-
-```yaml
-envoy:
-  tolerations:
-    - effect: NoSchedule
-      key: scalar-labs.com/dedicated-node
-      operator: Equal
-      value: scalardb
+      preferredDuringSchedulingIgnoredDuringExecution:
+        - podAffinityTerm:
+            labelSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                    - scalardb-cluster
+                - key: app.kubernetes.io/app
+                  operator: In
+                  values:
+                    - envoy
+            topologyKey: kubernetes.io/hostname
+          weight: 50
```

Comment:
Instead of `requiredDuringSchedulingIgnoredDuringExecution (too strict recommendation)`, we use `preferredDuringSchedulingIgnoredDuringExecution (slightly looser constraints)` to spread pods on the Kubernetes cluster.

This new configuration doesn't prevent the auto-scale of pods (It can scale pods without node scale).